projects/js-packages/base-styles/gutenberg-base-styles.scss

```diff
@@ -1,6 +1,6 @@
-@import '@wordpress/base-styles/z-index';
-@import '@wordpress/base-styles/colors';
-@import '@wordpress/base-styles/variables';
-@import '@wordpress/base-styles/breakpoints';
-@import '@wordpress/base-styles/mixins';
-@import '@wordpress/base-styles/animations';
+@import '~@wordpress/base-styles/z-index';
```

Comment:
I've had to add the tildes here as Boost is using Rollup, and it doesn't resolve the scss files from `node_modules` without it.

p1651747887163089-slack-CDLH4C1UZ